### ISSUE TITLE
Editor's plugins prop isn't parsed as expected 

### ISSUE THREAD

**Issue by youweol-master** [tinymce/tinymce-react#347]
@tinymce/tinymce-react: "^4.1.0"
tinymce version: 6.0.3

code like 
```
<Editor
    // omitted
   plugins: [
            'advlist autolink lists link image charmap preview anchor',
            'searchreplace visualblocks code fullscreen',
            'insertdatetime media table code help wordcount'
          ],
 />
```

will cause errors like 
```
Failed to load plugin: advlist autolink lists link image charmap preview anchor from url plugins/advlist autolink lists link image charmap preview anchor/plugin.min.js
```
in chrome console.

it seems that the editor treat the whole string in plugins field as one plugin name.

code like this will fix this issue.
```
<Editor
    // omitted
 plugins: [
            ...'advlist autolink lists link image charmap preview anchor'.split(" "),
            ...'searchreplace visualblocks code fullscreen'.split(" "),
            ...'insertdatetime media table code help wordcount'.split(" ")
          ],
  />
```

Is that a BUG or mismatched tinymce and tinymce-react version？

**Comment by tiny-james** [comment]
The react integration will accept either a single string or an array of strings. If it gets a single string it will split it on space (note it does not currently support comma separated). If it gets an array of strings it will assume they are already separate plugins.

It then passes the parsed plugin list as an array to TinyMCE itself. This is needed because it combines the values in the `init.plugins` prop with the values in the `plugins` prop.

It's worth noting that TinyMCE 6 does not support the "array of space separated strings" option though that was supported in TinyMCE 5.
https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#plugins

So in summary this is working as intended but the integration could possibly support comma separated lists of plugins in the future to match TinyMCE 6. The integration is not likely to ever support arrays of strings that need further splitting because TinyMCE 6 does not support it.

**Comment by tiny-james** [comment]
Note that the reason this might have worked for TinyMCE 5 is that it would have split the strings after it received an array of strings from the integration.
